templates/workloads/partials/cf/state-machine.yml

```diff
@@ -38,6 +38,9 @@ StateMachine:
             {{- range $sg := .Network.SecurityGroups }}
             - {{$sg}}
             {{- end }}
+            {{- if .NestedStack}}{{$stackName := .NestedStack.StackName}}{{range $sg := .NestedStack.SecurityGroupOutputs}}
```

Comment:
Do we need to update any of our local-integ tests to capture this change?

Reply:
It seems like we currently don't have any existing local-integ test for workload deployed with an addon (I could be wrong), so this change is not captured in tests.

I agree that it'd be nice if we can implement some integ-tests to check whether the addon outputs is properly rendered into the workload template, given an addon template file and a workload object. 

Right now it seems like it's a little difficult to isolate an addon file in order to do that. Seems like we would need to  set up a dummy `/copilot` directory that contains a `/<workload>/addons` directory.